cg/meta/upload/nipt/models.py

```diff
@@ -6,3 +6,18 @@ class StatinaUploadFiles(BaseModel):
     result_file: str
     multiqc_report: Optional[str]
     segmental_calls: Optional[str]
+
+
+class FlowCellQ30AndReads(BaseModel):
+    """Summary of a flow cell"""
+
+    total_reads_on_flow_cell: int
+    average_q30_across_samples: float
+
+    def reads_above_threshold(self, threshold: int) -> bool:
+        """Check if the number of reads is above a threshold"""
+        return self.total_reads_on_flow_cell >= threshold
```

Comment:
I think this sounds more like more like a bool return
```suggestion
    def passes_read_threshold(self, threshold: int) -> bool:
        """Check if the number of reads is above a threshold."""
        return self.total_reads_on_flow_cell >= threshold
```